manifests/certonly.pp

```diff
@@ -145,6 +160,7 @@
     }
   } else {
```

Comment:
When `$manage_cron` is set to `false` this means that cron should not be managed.
And so `else` is a nonsense. 

IMO, the `if $manage_cron` block should be moved in the code that handle `$ensure` with two differents  `if $manage_cron` blocks. A first one that explain what to do when `$ensure` is `present` and one other when `$ensure` is `absent'.

Reply:
Good call. I'll be implementing this next.